deployments/nimbus-k8tls/templates/daemonset.yaml

```diff
@@ -21,4 +21,11 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: K8TLS_NAMESPACE
+            value: {{ include "nimbus-k8tls.fullname" . }}-env
```

Comment:
I don't think we need `K8TLS_NAMESPACE` env variable. We can use `nimbus-k8tls-env` hardcoded namespace to deploy all k8tls related stuff.